Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -37,11 +39,470 @@ public IEnumerator TestAddInteractableAtRuntime()
+                p.LookAt(Vector3.forward);
+            });
+
+            Vector3 targetStartPosition = translateTargetObject.localPosition;
+
+            yield return null;
+
+            // Find the menu action from the input system profile
+            MixedRealityInputAction menuAction = MixedRealityToolkit.InputSystem.InputSystemProfile.InputActionsProfile.InputActions.Where(m => m.Description == "Menu").FirstOrDefault();
+            Assert.NotNull(menuAction.Description, "Couldn't find menu input action in input system profile.");
+
+            // Set the interactable to respond to a 'menu' input action
+            interactable.InputAction = menuAction;
+
+            // Find an input source to associate with the input event (doesn't matter which one)
+            IMixedRealityInputSource defaultInputSource = MixedRealityToolkit.InputSystem.DetectedInputSources.FirstOrDefault();
+            Assert.NotNull(defaultInputSource, "At least one input source must be present for this test to work.");
+
+            // Raise a menu down input event, then wait for transition to take place
+            MixedRealityToolkit.InputSystem.RaiseOnInputDown(defaultInputSource, Handedness.Right, menuAction);
+            yield return new WaitForSeconds(1f);
+
+            Assert.AreNotEqual(targetStartPosition, translateTargetObject.localPosition, "Transform target object was not translated by action.");
+
+            // Raise a menu up input event, then wait for transition to take place
+            MixedRealityToolkit.InputSystem.RaiseOnInputUp(defaultInputSource, Handedness.Right, menuAction);
+            yield return new WaitForSeconds(1f);
+
+            Assert.AreEqual(targetStartPosition, translateTargetObject.localPosition, "Transform target object was not translated back by action.");
+
+            Object.Destroy(interactableObject);
+            // Wait for a frame to give Unity a change to actually destroy the object
+            yield return null;
+        }
+
+        /// <summary>
+        /// Instantiates a push button prefab and uses simulated voice input events to press it.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator TestSimulatedVoiceInputOnPrefab()
+        {
+            // instantiate scene
+            TestUtilities.InitializeMixedRealityToolkitAndCreateScenes(true);
+            TestUtilities.InitializePlayspace();
+
+            // Load interactable prefab
+            GameObject interactableObject;
+            Interactable interactable;
+            Transform translateTargetObject;
+
+            InstantiateDefaultInteractablePrefab(
+                new Vector3(0.0f, 0.0f, 0.5f),
+                new Vector3(-90f, 0f, 0f),
+                out interactableObject,
+                out interactable,
+                out translateTargetObject);
+
+            // Set up its voice command
+            interactable.VoiceCommand = "Select";
```

Comment:
I think after #4695 you will need to update this to actually make sure that the button is being focused on, otherwise the test may fail. There's a new checkbox "requires focus" that might be checked in the model_pushbutton prefab. It will ensure that the object is actually being looked at before allowing interactable to activate from voice command.